src/widget/wtracktableview.cpp

```diff
@@ -755,14 +755,25 @@ TrackModel* WTrackTableView::getTrackModel() const {
 }
 
 void WTrackTableView::keyPressEvent(QKeyEvent* event) {
-    if (event->key() == Qt::Key_Return) {
-        // It is not a good idea if 'key_return'
-        // causes a track to load since we allow in-line editing
-        // of table items in general
-        return;
-    } else {
-        QTableView::keyPressEvent(event);
+    // If Ignore all Enter combos if any cell editor is open
+    if (event->key() == Qt::Key_Return &&
```

Comment:
unlike many other keys Qt::Key_Return does not reach this function it is already used for  
[PreviewDeck1],start_stop

Reply:
oh. I had a custom kbd config in place which didn't contain this shortcut. also it does not work when I have **no** kbd config in .mixxx that contains it and start mixxx from the build directory. strange..
And it needs to be added to the keyboard shortcut PDF/image.

Anyway, then let's not use it. We could argue if `Return` is maybe better used for the double click action, not for the preview deck, but that doesn't need to happen here.

Reply:
> We could argue if Return is maybe better used for the double click action

I can confirm that.